LDKNodeMonday/View/Home/NetworkSettingsView.swift

```diff
@@ -10,57 +10,111 @@ import SwiftUI
 
 struct NetworkSettingsView: View {
     @Environment(\.dismiss) private var dismiss
+    @EnvironmentObject var viewModel: NetworkSettingsViewModel
 
-    @EnvironmentObject var viewModel: OnboardingViewModel
+    @State private var showRestartAlert = false
+    @State private var tempNetwork: Network?
+    @State private var tempServer: EsploraServer?
```

Comment:
I think we can figure out a different way to do this rather than these temporary values.

As a start to make sure I follow are they just holding state between a picker change and an alert confirmation?

Reply:
Happy for suggestions to avoid.
Since one or both of the values are needed in the `.alert` they need to be stored somewhere.
If the name temp is the objectionable part with can use newNetwork / newServer instead :)

Reply:
yeah makes sense, and works so I'm good w it